### PR TITLE
Adds fly wings, fixes robo & skeleton wings

### PR DESCRIPTION
**Pull request by Sheits**
## About The Pull Request

Adds fly wings and fixes the android and skeleton wings that were merged recently. Also on a side note the skeleton wings are missing 3/4 of their down pose sprites but I don't really wanna go look for where the guy who originally ported them got them.

## Why It's Good For The Game

I'm not really sure what to say, a few people asked me to make this and it fixes existing issues with the other wings.
image

## Changelog
:cl:
fix: Made android and skeleton wings work instead of telling the user "you feel nothing but a terrible aftertaste" when they drink strange elixir
fix: Made android and skeleton wings not use a mutant colour overlay like lizard wings
imageadd: Fly wings :)
code: Added android species check
/:cl:

### CODE REVIEW

**Comment by Sheits** [comment]
Just a heads up I wont be able to look at this for about another 12+ hours

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -1852,6 +1852,7 @@
 /datum/sprite_accessory/wings/skeleton
 	name = "Skeleton"
 	icon_state = "skele"
+	color_src = 0
```
Comment (Fikou) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:1855]: use boolean defines here, FALSE instead of 0

**Comment by Nobelium-XVIII** [comment]
Now make plasmamen wings

**Comment by tralezab** [comment]
thanks

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -629,7 +629,7 @@
 	if(iscarbon(exposed_mob) && exposed_mob.stat != DEAD)
 		var/mob/living/carbon/exposed_carbon = exposed_mob
 		var/holycheck = ishumanbasic(exposed_carbon)
-		if(reac_volume < 5 || !(holycheck || islizard(exposed_carbon) || (ismoth(exposed_carbon) && exposed_carbon.dna.features["moth_wings"] != "Burnt Off"))) // implying xenohumans are holy //as with all things,
+		if(reac_volume < 5 || !(holycheck || isflyperson(exposed_carbon) || isskeleton(exposed_carbon) || isandroid(exposed_carbon) || islizard(exposed_carbon) || (ismoth(exposed_carbon) && exposed_carbon.dna.features["moth_wings"] != "Burnt Off"))) // implying xenohumans are holy //as with all things,
```
Comment (tralezab) [inline code/modules/mining/lavaland/necropolis_chests.dm:632]: remove this from this pr, i'll make it atomized by removing this retarded check in the first place

Comment (tralezab) [reply code/modules/mining/lavaland/necropolis_chests.dm:632]: #59756

Comment (Sheits) [reply code/modules/mining/lavaland/necropolis_chests.dm:632]: sure

**Comment by Sheits** [comment]
I think this did it, lmk if it didn't

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 1 above)
Comment (Sheits) [reply code/modules/mob/dead/new_player/sprite_accessories.dm]: done

**Review by Rohesie — APPROVED** [review]
Code is fine. It now depends on the art nerds.

**Pull request merged**